pages/_document.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Document, { Html, Head, Main, NextScript } from 'next/document'
+
+class MyDocument extends Document {
+	render() {
+		return (
+			<Html>
+				<Head />
+				<body>
+					<script src="noflash.js" />
```

Comment:
I don't know if the file source is going to break on pages other than root.

Reply:
@eduardoPinto12 ☝️.

Reply:
I didn't fully understand this, are you suggesting to move it to the root?

Reply:
Not moving to the root, linking to the root like this: https://github.com/ruisaraiva19/favycon/pull/21/files#diff-8f3a5cf5019173ef1f56a54e45e44747R10